src/services/services.ts

```diff
@@ -592,7 +592,7 @@ namespace ts {
      * @returns `true` if `node` has a JSDoc "inheritDoc" tag on it, otherwise `false`.
      */
     function hasJSDocInheritDocTag(node: Node) {
-        return getJSDocTags(node).some(tag => tag.tagName.text === "inheritDoc");
+        return getJSDocTags(node).some(tag => tag.tagName.text === "inheritDoc" || tag.tagName.text === "inheritdoc");

```

Comment:
jsdocs uses the lowercased version while javadoc uses the camel-cased version. Both should be supported?

https://jsdoc.app/tags-inheritdoc.html
https://docs.oracle.com/javase/6/docs/technotes/tools/solaris/javadoc.html#@inheritDoc